Ada Examples/Examples Based On OpenGL Superbible/Simplest/just_a_dot.gpr

```diff
@@ -0,0 +1,194 @@
+limited with "../../../Ada_Source/OpenGLAda-master/opengl-glfw.gpr";
+
+project Just_A_Dot is
+
+   for Source_Dirs use ("src");
+   for Object_Dir use "obj/";
+   for Exec_Dir use ".";
+   type Library_Kinds is
+      ("static", "relocatable");
+   Library_Type : Library_Kinds := external ("LIBRARY_TYPE");
+   type Toggle_Type is
+      ("enabled", "disabled");
+   Auto_Exceptions : Toggle_Type := external ("Auto_Exceptions");
+   type Mode_Type is
+      ("debug", "release");
+   Mode : Mode_Type := external ("Mode");
+
+   -- Different operating systems need different linker
+   -- flags. Moreover, some features (such as WGL, GLX,
+   -- CGL) are only available on one specific platform.
+   -- Supported values:
+   --  * windows : Microsoft Windows
+   --  * x11     : X Window System
+   --  * quartz  : Quartz Compositor (OS X)
+   type Windowing_System_Type is
+      ("windows", "x11", "quartz");
+   Windowing_System : Windowing_System_Type := external ("Windowing_System");
+   type Glfw_Version_Type is
+      ("2", "3");
+   Glfw_Version : Glfw_Version_Type := external ("GLFW_Version");
+
+   case Library_Type is
```

Comment:
OK!
I must have initially copied the gpr file from somewhere.
I deleted that one and have generated a new one, from which I intend to delete Ada_Source/OpenGLAda/ before submitting:
limited with "../../../Ada_Source/OpenGLAda/opengl-glfw.gpr";
project Just_A_Dot is
   for Source_Dirs use ("src");
   for Object_Dir use "obj/";
   for Exec_Dir use ".";
   for Main use ("a_dot.adb");
end Just_A_Dot;